ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/IntegrationTestBase.java

```diff
@@ -105,12 +106,13 @@
 
   @TempDir static Path dataPath;
 
-  static void setupEthSigner(final long chainId) throws IOException, CipherException {
+  static void setupEthSigner(final long chainId)
+      throws IOException, CipherException, ExecutionException, InterruptedException {
     setupEthSigner(chainId, "");
   }
 
   static void setupEthSigner(final long chainId, final String downstreamHttpRequestPath)
-      throws IOException, CipherException {
+      throws IOException, CipherException, ExecutionException, InterruptedException {
```

Comment:
nit: perhaps just throw Exception not much benefit in declaring all the exceptions this can throw to simplify usage in tests. we are never going to deal with them separately in the tests